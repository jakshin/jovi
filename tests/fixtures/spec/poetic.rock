(Poetic constant literals)
My heart is true
  Say my heart

Tommy is nobody
  Say Tommy

Tommy is mysterious
  Say Tommy

(Poetic string literals)
Peter says Hello San Francisco!
  Say Peter

San Francisco says Hello back
  Say San Francisco

(Poetic number literals)
Tommy was a lovestruck ladykiller (100)
  Say Tommy

Sweet Lucy was a dancer (16)
  Say Sweet Lucy

A killer is on the loose (235)
  Say a killer

My dreams were ice. A life unfulfilled; wakin' everybody up, taking booze and pills (3.1415926535)
  Say my dreams

Tommy was without (7)
  Say Tommy
