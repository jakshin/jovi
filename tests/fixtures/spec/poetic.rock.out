true
null
mysterious
Hello San Francisco!
Hello back
100
16
235
3.1415926535
7
